index.html

```diff
@@ -648,25 +649,31 @@ <h3>Additional Properties</h3>
           <h3>Versioning</h3>
           <p>
             Versioning is not provided as an explicit feature of this specification.
-            It is advised to make backwards compatabile changes to schemas, should
-            they be adjusted. Otherwise, it is advised that a new <a>credential schema</a>
-            is created with a unique identifier.
-          </p>
-          <p class="issue" data-number="120">
-            Add language on enabling schema immutability/versioning, such as making use of hashlinks.
+            Implementers are advised to make backwards compatabile changes to schemas, should
+            they be adjusted. Otherwise, it is advised that new <a>credential schemas</a>
+            are created with unique identifiers to avoid processing conflicts.
           </p>
         </section>
+        <section class="informative">
+         <h3>Content Integrity Protection</h3>
+         <p>
+            It is important to make sure that <a>credential schemas</a> have not been tampered with
+            before processing. When making use of the <code>JsonSchemaCredential2023</code> representation
+            of a schema, the credential's associated integrity protection mechanism can act
+            as mutability detection since digital signatures break with content changes.
+         </p>
+         <p>
+            As an alternative, the aforementioned [[SRI]] scheme may be used to
+            provide content integrity protection, ensuring that the underlying <a>credential schema</a>
+            resource has not been tampered with.
```

Comment:
> An issuer of the credential schema may modify the contents of the JsonSchemaCredential, and integrity protection would still pass

If they re-sign the credential this is true, but not otherwise which is the main concern...

I think this raises a larger question -- are VCs immutable? and I'm not sure...I think the answer is no. The secured form of these credentials are immutable at least

Reply:
My concern is the verifiers need the ability to run validation with the same data that the original credential was issued with. Otherwise a credential's validity may change unexpectedly.

> I think this raises a larger question -- are VCs immutable? and I'm not sure...I think the answer is no. The secured form of these credentials are immutable at least

That's a good question. I'm not sure either, I also lean towards no. The secured form is immutable, but it's easy to mutate the value that results from dereferencing a `@id` by simply changing what the URL points to.

My suggestion is to let implementers know that they should:
* Use the `digestSRI` in the original credential when they want to ensure that the contents have not been changed since issuance.
* Not use it and accept that they're trusting the publisher of the schema to mutate it in a backwards compatible way.